### PR TITLE
docs: korean translation

### PR DESCRIPTION
**Pull request by kijunkim9**
korean translation for controls

### CODE REVIEW

File: docs/examples/ko/controls/DeviceOrientationControls.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="ko">
```
Comment (Mugen87) [inline docs/examples/ko/controls/DeviceOrientationControls.html:2]: Shouldn't this be `kr`, see #20872? I'm not sure what shortcut is right...

Comment (kijunkim9) [reply docs/examples/ko/controls/DeviceOrientationControls.html:2]: Isn't `kr` the country code? I think the language code `ko` is correct.

Comment (Mugen87) [reply docs/examples/ko/controls/DeviceOrientationControls.html:2]: @hareha What do you think?

Comment (Mugen87) [reply docs/examples/ko/controls/DeviceOrientationControls.html:2]: I've checked the ISO standard and indeed it is `ko`!

https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=ko

**Comment by Mugen87** [comment]
BTW: Please do not copy documentation pages which are not yet translated. As a result of #20098, it's better not to do this and have a 404 page instead

**Comment by kijunkim9** [comment]
I updated It .

**Comment by Mugen87** [comment]
Now that the  #20872 was merged, can you please resolve the merge conflicts in `list.json`?  Besides, please include only links to new translated files. Right now, there are still way too many additions in that file.

File: docs/examples/ko/controls/FirstPersonControls.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		<base href="../../../" />
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+
+		<h1>[name]</h1>
+
+		<p class="desc">
+			This class is an alternative implementation of [page:FlyControls].
```
Comment (Mugen87) [inline docs/examples/ko/controls/FirstPersonControls.html:14]: This sentence still needs a translation.

File: docs/list.json [hunk 3]
```diff
@@ -999,8 +999,8 @@
 		}
 
 	},
-
-	"ko": {
+
```
Comment (Mugen87) [inline docs/list.json:1002]: Please do not change the formatting of the file.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**